app/services/force/event_subscriber_translate_service.rb

```diff
@@ -19,6 +19,10 @@ class EventSubscriberTranslateService
     def initialize
       setup_salesforce_client
       setup_faye_client
+      @translation_service = GoogleTranslationService.new(
```

Comment:
Not sure if we want to / need to initialize GoogleTranslationService here.

Reply:
Yes, it would be best to follow the initialization pattern of other services in this file.